share/completions/wpctl.fish

```diff
@@ -38,6 +42,18 @@ function __wpctl_command_shape
     end
 end
 
+function __wpctl_get_settings
+    set -l wpctl_settings (wpctl settings | string collect)
```

Comment:
should maybe silence the error on old versions (`wpctl settings 2>/dev/null`) though I guess it's not a big deal

Reply:
wpctl seems to not print to sderr when an unknown command is called, instead it print a help message and return exit status 1.

Just to be sure, I tested it now on ubuntu 24 that uses wireplumber 0.4.17, using fish 4+, and I didn't notice any unexpected behavior.